### PR TITLE
📋 Introduce documentation, license, code of conduct etc

### PR DESCRIPTION
**Pull request by marekkirejczyk**

### CODE REVIEW

File: docs/source/_static/custom.css [hunk 1]
```diff
@@ -0,0 +1,251 @@
+html {
```
Comment (rzadp) [inline docs/source/_static/custom.css:1]: Is this ours or is it copied from somewhere?
If copied can we have a link or info where this is coming from?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Copyright 2020 Ethworks sp z o.o.
```
Comment (rzadp) [inline LICENSE:1]:
```suggestion
MIT License

Copyright (c) 2020 Ethworks sp z o.o.
```

File: packages/core/README.md [hunk 3]
```diff
@@ -50,9 +50,44 @@ function App() {
...
+[Install Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html) to build documentation:
+
+```sh
+cd docs
+make html
+```
+
+Before building documentation for the first time you may have to install required python packages:
+```sh
+pip install -r docs/requirements.txt
```
Comment (rzadp) [inline packages/core/README.md:88]: Looks like Sphinx is not working with python 2.7. Should we say `pip3` to indicate that python3 is required?

```suggestion
pip3 install -r docs/requirements.txt
```

**Review by rzadp — APPROVED** [review]
Left a couple of suggestions.

File: docs/source/_static/custom.css (hunk 1 above)
Comment (marekkirejczyk) [reply docs/source/_static/custom.css:1]: Form Waffle :)

**Pull request merged**